omero/sysadmins/cli/data-ownership.txt

```diff
@@ -0,0 +1,53 @@
+Changing the ownership of data
+==============================
+
+As described in :doc:`../server-permissions` every imported plate or
+image is in a particular group and owned by a particular user. OMERO
+administrators must sometimes move data to a different group and/or give
+it to a different user.
+
+.. warning::
+
+  Administrators may move data into a group of which the data owner is
+  not a member. Also, administrators may give data to a user who is not
+  a member of the data's group. These actions may be reasonable steps in
+  a larger workflow but do note the potential to confuse users who find
+  that they own data that is not in any of their groups.
```

Comment:
I think this is a little confusing. Here's my attempt at it...
"Data does not need to be assigned to a group where the data owner is a member, and administrators may wish to change the ownership of data or move it between groups in several steps of a larger workflow. However, it is generally expected that data should end up in a group where the data owner is a member, so that they can view their data in the OMERO clients."